<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="User" failures="1" tests="3" name="ListeningAllTest" time="0.146" errors="0" timestamp="2020-10-20T23:35:01 EDT">
  <testcase classname="testNGConcept.TestingListenerConcpet" name="Test2" time="0.028">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNGConcept.TestingListenerConcpet.Test2(TestingListenerConcpet.java:20)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase classname="testNGConcept.TestingListenerConcpet" name="Test1" time="0.002"/>
  <testcase classname="testNGConcept.TestingListenerConcpet" name="Test3" time="0.001">
    <skipped/>
  </testcase> <!-- Test3 -->
</testsuite> <!-- ListeningAllTest -->
